templates/default.nginx_http.conf.template

```diff
@@ -15,7 +15,13 @@ log_format  custom  '$http_x_forwarded_for - $remote_user [$time_local] "$reques
 # workaround for the way we're including conf https://serverfault.com/questions/729128/overriding-nginx-access-log-directive-duplicate-log-entries
 access_log  off;
 
+# nginx caches ips on startup, so doesn't survive webservice restarts
+# https://www.nginx.com/blog/dns-service-discovery-nginx-plus/
+# https://stackoverflow.com/questions/46660436/nginx-does-not-automatically-pick-up-dns-changes-in-swarm/46664433#46664433
+resolver 127.0.0.11 valid=10s;
```

Comment:
I guess the trade-off here is that constantly re-resolving can slow things down when the webservice does not go down. Do you have any idea of the impact of this on performance? Presumably it's fine and worth the trade-off, but just want to make sure.

Reply:
Not sure, but my thought is that re-resolving every 10 seconds internally shouldn't be too costly, but it would be good to run performance tests and/or test staging